pkg/storageos/rbac.go

```diff
@@ -41,6 +41,9 @@ const (
 	SchedulerClusterRoleName    = "storageos:scheduler-extender"
 	SchedulerClusterBindingName = "storageos:scheduler-extender"
 
+	SchedulingClusterRoleName    = "storageos:scheduling-extender"
```

Comment:
`Scheduler` for kube-scheduler and `Scheduling` for the extender could be confusing.
How about `SchedExt` or `SchedulerExt`?

Reply:
The suggested names are also confusing for me. Which one is which? Let me find some better name for them.